main.tf

```diff
@@ -197,8 +197,4 @@ resource "aws_ecs_service" "default" {
     container_name   = "${var.container_name}"
     container_port   = "${var.container_port}"
   }
-
-  lifecycle {
-    ignore_changes = ["task_definition"]
```

Comment:
No, this is not good. It's compatible with CI/CD with CodeBuild/CodePipeline. If we use CD in CodePipeline, the task definition will change. A subsequent apply by terraform will trigger a rollback.

Reply:
ok will put it back.
just very difficult to test it

Reply:
Yea, agree. It's unfortunate that interpolations are not supported in `lifecycle` blocks.